app/scripts/modules/core/src/managed/overview/artifact/Artifact.less

```diff
@@ -0,0 +1,117 @@
+.Artifact {
+  .artifact-current-version {
+    display: flex;
+    flex-direction: column;
+    border-radius: 5px;
+  }
+
+  .artifact-versions-title {
+    margin-bottom: var(--s-spacing);
+    text-transform: uppercase;
+    color: var(--color-titanium);
+    font-size: 14px;
+    font-weight: 600;
+  }
+
+  .artifact-pending-versions {
+    margin-top: var(--l-spacing);
+    position: relative;
+    --circle-shift: 6px;
+
+    .artifact-pending-versions-list {
+      padding-left: var(--s-spacing);
+
+      .artifact-pending-version {
+        display: flex;
+        flex-direction: column;
+        padding-right: var(--l-spacing);
+        border-radius: 5px;
+        position: relative;
+        padding-left: 48px;
```

Comment:
For padding and margin, you can use some of the stylguide classes `sp-padding-l-right`. For flex items, you can use `horizontal` and then add other classes for placement (middle, center, etc). More information is in the styleguide.

Reply:
I need to get used to it. I think that we should consider using TailwindCSS - https://tailwindcss.com/
It's really awesome